### DISCUSSION TITLE
Contrast   - Defining, measuring, and provision wording

### DISCUSSION THREAD

**Discussion by GreggVan** [w3c/wcag3#14] [General]
There has been discussion about changing the CONTRAST language, measures, etc.  Some of this may be due to better knowledge,  some of it to changing technologies.   But,from discussions so far, it is clearly complicated .  So this discussion is created to capture all the different information and opinions.   Suggestion: that we separate papers and research etc. posts (source posts) from opinion posts (what we think the "answer" for WCAG might be.

**Comment by rachaelbradley** [comment]
We will capture all research at [Research at Color Contrast](https://github.com/w3c/wcag3/issues/10) and review and organize the research as part of the subgroup work on this guideline.
